diff --git a/bin/index.js b/bin/index.js
index 72bbe88a99da92147bab047eac6edad4aa42060a..feec0098f2d0e3673a7ca92e8c9f753d50bf6df2 100644
--- a/bin/index.js
+++ b/bin/index.js
@@ -680,7 +680,7 @@ function emit(rootDecl, _a) {
         print(d.name);
         writeTypeParameters(d.typeParameters);
         if (d.baseTypes && d.baseTypes.length) {
-            print("extends ");
+            print(" extends ");
             var first = true;
             for (var _i = 0, _a = d.baseTypes; _i < _a.length; _i++) {
                 var baseType = _a[_i];
